metrics/validate_drp.yaml

```diff
@@ -0,0 +1,223 @@
+# validate_drp metric definitions
+# Definitions from LPM-17 (the Science Requirements Document).
+# http://ls.st/lpm-17
+
+PA1:
+  description: >
+    The maximum rms of the unresolved source magnitude distribution around the
+    mean value.
+  unit: mmag
+  reference:
+    doc: LPM-17
+    url: http://ls.st/lpm-17
+    page: 21
+
+PF1_minimum_gri:
+  description: >
+    The maximum fraction of magnitudes deviating by more than PA2_minimum_gri
+    from mean (%).
+  unit: ''
+  reference:
+    doc: LPM-17
+    url: http://ls.st/lpm-17
+    page: 21
+
+PF1_design_gri:
```

Comment:
So, PF1 and PA2 are complicated:

> No more than PF1 % of the measurements will deviate by more than PA2 millimag from the mean

So to measure PF1, you need to know PA2, and vice versa. Our approach to this is to use the *specification* value for one when measuring the other. But it means that now we have a new metric per specification level of each other's metrics. This is definitely a corner case and I hope that most metrics aren't written this way.